UnityProject/Assets/Scripts/Objects/Singularity.cs

```diff
@@ -427,7 +428,7 @@ private void DestroyObjects(IEnumerable<Vector3Int> coords)
 						if (health.RegisterPlayer.PlayerScript.Mind?.occupation == OccupationList.Instance.Get(JobType.CLOWN))
 						{
 							health.OnGib();
-							ChangePoints(DMMath.Prob(50) ? -1000 : 1000);
+							ChangePoints(Random13.ProbFromTime(GameManager.Instance.RoundTime) ? -1000 : 1000);
```

Comment:
I don't understand when to use the Prob and ProbFromTime.

In the original code this was 50% chance. What's the new chance of occurrence?

Reply:
so how do you make something happen 5% of the time?

Reply:
Use `ProbChance()` 
I just commited it